webapp/src/app/components/dialogs/confirm-close/confirm-close.component.html

```diff
@@ -0,0 +1,16 @@
+<cc-overlay-panel 
+  cdkTrapFocus [cdkTrapFocusAutoCapture]="true" 
+  title="You have unsaved changes"
+  [customSize]="{width: '30em', height: 'auto'}"
+  >
+	<ng-container ngProjectAs="content">
+    <div class="p-3 text-center">
+      Are you sure you want to discard your changes?
+    </div>
+	</ng-container>
+	<ng-container ngProjectAs="actions">
+    <button mat-stroked-button (click)="close(false)">Go back</button>
+    <button mat-stroked-button (click)="close(null)" *ngIf="showDevMode && devMode" cdkFocusInitial>DEV: Reload</button>
+    <button mat-stroked-button (click)="close(true)" cdkFocusInitial>Discard changes</button>
```

Comment:
I think it would be better to have the initial focus be on "Go back". It's not clear if the user really wanted to discard everything or just forgot to save, so we should default on the harmless option